packages/server/src/sdk/app/views/index.ts

```diff
@@ -31,14 +35,62 @@ export async function getEnriched(viewId: string): Promise<ViewV2Enriched> {
   return pickApi(tableId).getEnriched(viewId)
 }
 
+async function guardViewSchema(
+  tableId: string,
+  viewSchema?: Record<string, ViewUIFieldMetadata>
+) {
+  if (!viewSchema || !Object.keys(viewSchema).length) {
+    return
+  }
+  const table = await sdk.tables.getTable(tableId)
+  if (viewSchema) {
```

Comment:
You don't need this condition, you check for `!viewSchema` above and return early.